ee/clickhouse/sql/funnels/funnel.py

```diff
@@ -22,3 +22,35 @@
 ORDER BY max_step {top_level_groupby} ASC
 ;
 """
+
+FUNNEL_PERSONS_SQL = """
+SELECT max_step {top_level_groupby}, count(1) as cnt, id
```

Comment:
What's possible inside `top_level_groupby` ? I can't imagine what could go in here, since the subquery only returns `max_step` and `id` ?

Also, a test for this bit would be nice :)

Reply:
Judging by `format_results`, `cnt` seems unused?

Reply:
It's a copy/paste thing. Removing.